### ISSUE TITLE
Create Fine-Tuner Tool for Competition Week Vision Refining

### ISSUE THREAD

**Issue by sah4jpatel** [SWAMP-Blimps/CatchingBlimp#35]
Simple Script or Program to frame-by-frame label data that is captured in the first practice session of the competition. This day-of fine-tuning would enable much more reliable and robust predictions. 

The hand-annotations would probably take 10-30 minutes depending on the amount of frames, however the method of extracting the sample of images to pull from in the training set is as follows:

1. record the video 
2. split video into frames
3. use t-SNE + KMEANS to cluster images into n clusters (100 or so)
4. pull image closest to the cluster center (n images)
5. hand-annotate n images
6. the annotations + images passed into yolo training process
7. the outputs weights file gets generated to upload to blimps

**Comment by sah4jpatel** [comment]
First part:
The tool should take in a path to the yolo dataset file and some training parameters, set up the training environment, and then train the correct model (customized-v10)

Second part:
It should be a docker container that is given a path to the trained .pt file that first exports the model to onnx and then converts it to rknn format

This could be a gui tool or a cli tool that has the capability to spin up a dedicated environment for training and conversion.
